Sources/Rendering/Core/CubeAxesActor/index.js

```diff
@@ -114,7 +114,7 @@ function vtkCubeAxesActorHelper(publicAPI, model) {
     offset,
     results
   ) => {
-    const value = model.renderable.get_tmAtlas().get(text);
+    const value = model.renderable.getTmAtlas().get(text);
     if (!value) {
```

Comment:
OK if you want to use that one then you need to add _tmAtlas back into the get list and you can then remove my get_tmAtlas method please. As is this will fail as there is no getTmAtlas method.

Reply:
Ah, I see. Sorry, seems we fixed the issue at the same time in different ways. Since yours is merged, I'll drop this change.